### ISSUE TITLE
Drop Tables seems flawed.

### ISSUE THREAD

**Issue by Frankenberryberry** [SunwellWoW/Sunwell-TBC-Bugtracker#689]
Decription: 
You gain the same items way too often.

How it works: 
In Skull Rock I got 14 identical white quality cloaks. Same thing happened during the "the demon seed" quest in the barrens. Humanoid drop tables seems off.

How it should work: 
There should be a greater variety in drop tables.

Source (you should point out proofs of your report, please give us some source):

**Comment by Jeansapin** [comment]
That's pretty much the same thing with Small "color" Pouch, in Ammen Fields in Azuremyst on the 
Mutated Root Lasher.

**Comment by nimeral** [comment]
IMO the most obvious issue is white armor dropping too often. All those Veteran Bracers/Cloaks and whatnot. 

My experience is Teldrassil/Darkshore only, but I feel it's a general (not mob-specific) issue, at least in these zones.

**Comment by Frankenberryberry** [comment]
> IMO the most obvious issue is white armor dropping too often. All those Veteran Bracers/Cloaks and whatnot.
> 
> My experience is Teldrassil/Darkshore only, but I feel it's a general (not mob-specific) issue, at least in these zones.

yeah well... I quit the server already. It's not even remotely ready for launch -.- Not coming back. RIP Sunwell
